receiver/redisreceiver/README.md

```diff
@@ -0,0 +1,89 @@
+# Redis Receiver
+
+### Overview
+
+The Redis receiver is designed to retrieve Redis INFO data from a single Redis
+instance, build metrics from that data, and send them to the next consumer at a
+configurable interval.
+
+Status: beta
+
+### Details
+
+The Redis INFO command returns information and statistics about a Redis
+server (see [https://redis.io/commands/info](https://redis.io/commands/info) for
+details).
+The Redis receiver extracts values from the result and converts them to open
+telemetry metrics. Details about the metrics produced by the Redis receiver
+can be found by browsing [metric_functions.go](metric_functions.go).
+
+For example, one of the fields returned by the Redis INFO command is
+`used_cpu_sys` which indicates the system CPU consumed by the Redis server,
+expressed in seconds, since the start of the Redis instance.
+
+The Redis receiver turns this data into a gauge...
+
+```go
+func usedCPUSys() *redisMetric {
+	return &redisMetric{
+		key:    "used_cpu_sys",
+		name:   "redis/cpu/time",
+		units:  "s",
+		mdType: metricspb.MetricDescriptor_GAUGE_DOUBLE,
+		labels: map[string]string{"state": "sys"},
+	}
+}
+```
+
+with a metric name of "redis/cpu/time" and a units value of "s" (seconds).
+
+# Configuration
+
+Note: this receiver is in beta and configuration fields are subject to change.
+
+Example configuration:
+
+```yaml
+receivers:
+  redis:
+    endpoint: "localhost:6379"
+    collection_interval: 10s
+    password: "s3cret"
+```
+
+### endpoint
+
+The hostname and port of the Redis instance, separated by a colon.
+
+_Required._
+
+### refresh_interval
```

Comment:
Is it collection_interval or refresh_interval?

Reply:
Should be collection interval -- I renamed the field and missed this file. Fixed.